controllers/grid/users/author/form/PKPAuthorForm.inc.php

```diff
@@ -231,10 +231,10 @@ public function execute(...$functionParams)
         parent::execute(...$functionParams);
 
         if ($existingAuthor) {
-            $authorDao->updateObject($author);
+            Repo::author()->edit($author, []);
```

Comment:
This one is had because there is a lot of code to refactor to use the params properly. Since we want to replace this form soon, you can leave this one as you've done it.

Reply:
How it will be replaced? What the new form will be?

Reply:
We hope to replace the author grid with a list panel, and when that happens the author form will interact with the REST API and use a differnet UI component.